lib/plugins/aws/lib/naming.js

```diff
@@ -242,6 +242,13 @@ module.exports = {
   getMethodLogicalId(resourceId, methodName) {
     return `ApiGatewayMethod${resourceId}${this.normalizeMethodName(methodName)}`;
   },
+  getValidatorLogicalId(resourceId, methodName) {
```

Comment:
We could add unit tests for those in `naming.test.js` if we're into that. Although those functions will be covered by other tests...

Reply:
Yeah, I wrote the test then created the naming functions so they're covered, but no harm in a specific unit test too :)